tests/views_integration/Tribe/Events/Views/V2/Partials/Month_View/Calendar_Body/Day/Calendar_Events/Calendar_Event/Tooltip/CostTest.php

```diff
@@ -4,10 +4,10 @@
 
 use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
 
-class CtaTest extends HtmlPartialTestCase
+class CostTest extends HtmlPartialTestCase
 {
 
-	protected $partial_path = 'month/calendar-body/day/calendar-events/calendar-event/tooltip/cta';
+	protected $partial_path = 'month/calendar-body/day/calendar-events/calendar-event/tooltip/cost';
```

Comment:
fix tests here to use `$this->get_mock_event()`, also fix tests to match the test names and cases of the other tests (list and day view)

Reply:
still missing case without cost